docker/Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM rastasheep/ubuntu-sshd:18.04
 
```

Comment:
I'm considering adding `apt-get update` here. @dagguh WDYT?

Reply:
We should do it.
Gotchas: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get